udtraek.py

```diff
@@ -100,6 +100,29 @@ def settlement(self, start_date='2000-01-01', end_date='2019-01-01'):
         return pd.DataFrame(results)
 
 
+    def multi_wdex(self, ncpus=None):
+        if ncpus is None:
+            ncpus = cpu_count() -1
```

Comment:
```suggestion
            ncpus = cpu_count() - 1
```
Læs gerne [PEP 8](https://www.python.org/dev/peps/pep-0008/).